aci/resource_aci_mgmtinboobzone.go

```diff
@@ -154,7 +154,7 @@ func getRemoteOOBManagedNodesZone(client *client.Client, dn string) (*models.OOB
 	}
 	mgmtOobZone := models.OOBManagedNodesZoneFromContainer(mgmtOobZoneCont)
 	if mgmtOobZone.DistinguishedName == "" {
-		return nil, fmt.Errorf("OoB Managed Nodes Zone %s not found", dn)
+		return nil, fmt.Errorf("OOB Managed Nodes Zone %s not found", dn)
```

Comment:
if we use inband should we also use full description and use Out Of Band?

Reply:
Ok